client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -84,6 +84,7 @@ object CapiModelEnrichment {
         tagExistsWithId("tone/analysis") -> AnalysisDesign,
         tagExistsWithId("tone/comment") -> CommentDesign,
         tagExistsWithId("tone/letters") -> LetterDesign,
+        tagExistsWithId("tone/obituaries") -> ObituaryDesign,
```

Comment:
I'd put this above stuff like comment  and analysis - I can't think of a situation where we'd want something to come through as a different category to Obituary, so we probably want that near the top

Reply:
Done